packages/cli/src/utils/enableAutomine.ts

```diff
@@ -12,24 +12,19 @@ type MiningMode =
       blockTime: number;
     };
 
-export type EnableAutomineResult = { reset: () => Promise<void> };
+export type EnableAutomineResult = undefined | (() => Promise<void>);
```

Comment:
Not a super strong opinion, but the thing i like about returning an object with named parameter is that it's self-documenting, whereas with a function i need to look at the source / docs to know what the function does.